cashu/wallet/cli/cli_helpers.py

```diff
@@ -134,17 +134,19 @@ async def get_mint_wallet(ctx: Context):
     if len(mint_balances) > 1:
         await print_mint_balances(ctx, wallet, show_mints=True)
 
-        mint_nr_str = (
-            input(f"Select mint [1-{len(mint_balances)}, press enter for default 1]: ")
-            or "1"
+        mint_nr_str = input(
+            f"Select mint [1-{len(mint_balances)}] or press enter for mint with largest balance: "
```

Comment:
Ideally, it should indicate here which mint (number) is meant by "largest balance". Otherwise it's a bit a shot in the dark.